lib/cc/engine/duplication.rb

```diff
@@ -21,13 +22,15 @@ def run
         languages.each do |lang|
           next unless language_supported?(lang)
```

Comment:
instead of calling `next`, what about having `languages` return only supported languages?